boards/m5stack/m5stack_atoms3_lite/doc/index.rst

```diff
@@ -27,7 +27,7 @@ It features the following integrated components:
 Supported Features
 ==================
 
-The Zephyr m5stack_atoms3_lite board configuration supports the following hardware features:
+The Zephyr ``m5stack_atoms3_lite`` board target supports the following hardware features:
```

Comment:
likely here too

Reply:
Thanks, I've added this to the opt-out list.